starter-packs/README.md

```diff
@@ -1,3 +1,8 @@
+| :exclamation:  Starter packs have moved | You can now find them here: https://registry.terraform.io/namespaces/honeycombio |

```

Comment:
@sgsharma @jharley  I wish I could've commented before this got approved and merged.  I would've suggested we say that Terraform module-based Starter Packs have moved, but there's other SPs that may live on here.

Reply:
#14 is open :)